chapter_convolutional-modern/alexnet_vn.md

```diff
@@ -164,7 +170,10 @@ The associated competition, dubbed the ImageNet Challenge pushed computer vision
 challenging researchers to identify which models performed best at a greater scale than academics had previously considered.
 -->
 
-*dịch đoạn phía trên*
+Năm 2009, bộ dữ liệu ImageNet được tung ra, thách thức các nhà nghiên cứu huấn luyện các mô hình với 1 triệu ví dụ hình ảnh, 1.000 ví dụ cho mỗi 1.000 lớp đối tượng khác nhau. 
+Các nhà nghiên cứu, dẫn đầu bởi Fei-Fei Li, người mà giới thiệu bộ dữ liệu này đã dựa trên công cụ Tìm kiếm hình ảnh của Google để chọn lọc ra trước các bộ dữ liệu hình ảnh lớn làm ứng cử viên cho mỗi lớp và sử dụng kênh cung cấp dịch vụ cộng động Mechanical Turk của Amazon, dùng người đánh nhãn để xác định từng hình ảnh xem nó có thuộc về các lớp có liên quan hay không. 
+Với quy mô lớn chưa từng có từ trước đến nay. 
+Cuộc thi đi liền với bộ dữ liệu này, với cùng tên gọi là ImageNet Challenge đã thúc đẩy sự phát triển nghiên cứu về thị giác máy tính và học máy tiến tới một bước mới, thách thức các nhà nghiên cứu tìm ra được mô hình nào thể hiện tốt nhất với một bộ dữ liệu có quy mô lớn hơn so với các bộ dữ liệu dành cho học thuật đã được đề cập ở trước. 
```

Comment:
hm nửa cuối câu này viết đầy đủ mình nghĩ sẽ là `to identify which models performed best at a greater scale than the scale which academics had previously considered.`, nên mình đề xuất dịch như dưới đây:
```suggestion
Cuộc thi đi liền với bộ dữ liệu này, được đặt tên là ImageNet Challenge đã thúc đẩy sự phát triển nghiên cứu thị giác máy tính và học máy, thách thức các nhà nghiên cứu tìm ra mô hình tốt nhất ở quy mô lớn hơn quy mô nghiên cứu học thuật trước đó. 
```